modules/localgov_services_status/tests/src/Functional/ServiceStatusMessageVisibilityTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Drupal\Tests\localgov_services_status\Functional;
+
+use Drupal\node\NodeInterface;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests localgov service status messages.
+ *
+ * @group localgov_services
+ */
+class ServiceStatusMessageVisibilityTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $profile = 'testing';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'localgov_services_status',
+  ];
+
+  /**
+   * Test necessary fields have been added.
```

Comment:
Please confirm you really meant it :)

Reply:
Thanks for the comments @Adnan-cds 

I do indeed mean to set the profile to testing and the theme to stark. Making tests as decoupled as possible from the whole LocalGov Drupal stack seems to me to be a good thing.

We don't have agreement on what and when we should set the theme and profile in tests. Sometimes it makes sense to use the LocalGov ones, sometime we want to test things works with any theme and profile.

Reply:
Err.. I asked about line 33 only :)

Reply:
Ha! I read this as @stephen-cox did when it was displayed in the issue view :-) Comment could be updated for sure.

Reply:
The email I received on this included a large block of code and I failed to read the docblock :roll_eyes: 
Should now be fixed.

Reply:
Thanks :)